src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './harUtils';
 export * from './recordedHarMiddleware';
 export * from './recorderHarMiddleware';
+export * from "./harFileUtils";
```

Comment:
I happen to need these utils too. This export does run afoul of the #1 , however so does the `./recorderHarMiddleware` so if you're going to split between 'browser only' and 'server' requests - then you could tidy it then.